packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -1,23 +1,35 @@
 <template>
-    <div class="certifaction-verification">
-        <VerificationDemo v-if="demo !== false" @verify-demo="verifyDemo" @dragging-demo-doc="onDraggingDemoDoc"/>
-
-        <div v-if="filteredVerificationItems.length" class="verification-item-list" ref="results">
-            <VerificationItem
-                v-for="verificationItem in filteredVerificationItems"
-                :key="verificationItem.hash"
-                :verificationItem="verificationItem"/>
-        </div>
+    <div class="certifaction-verification"
+         @dragover.prevent="dragOver"
+         @dragleave="dragLeave"
+         @drop="handleDrop">
 
         <VerificationDropBox
-            @files-dropped="verify"
+            :class="{hidden: !dropbox.draggingOver}"
```

Comment:
Is there a reasy why you use css instead of v-if to hide/show these elements?

Reply:
Yes, the transition is smoother. When using v-if the single elements are popping up with a delay (especially the image)

Reply:
Ok, that makes sense. There is also v-show which uses css to show/hide. If you want to do it with css classes I would recommend to set a class like "drag-over" on the parent element and then use that class to show/hide the child elements.

Reply:
Solved it now with v-show